include/dev/wddr/phy_defs.h

```diff
@@ -169,15 +173,15 @@ typedef enum wddr_msr_t
  *
  * @details DFI frequency to DRAM frequency ratio.
  *
- * 1TO1     DRAM tCK = 1 * DFI tCK
- * 1TO2     DRAM tCK = 2 * DFI tCK
- * 1TO4     DRAM tCK = 4 * DFI tCK
+ * 1TO1     DRAM tCK = (1 << 0) * DFI tCK
+ * 1TO2     DRAM tCK = (1 << 1) * DFI tCK
+ * 1TO4     DRAM tCK = (1 << 2) * DFI tCK
```

Comment:
We should update these equations, I think. I think it's easier if we put that 1 DFI cycle equals (1 << ratio) DRAM cycles. I think what we have now is correct if you think in terms of frequency. But I think it's important to relay cycle relationship instead.

Reply:
Updated the text to indicate relationship and not be an equation.